### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by omahs**
Fix typos

### CODE REVIEW

File: docs/guides/dao-members/staking.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ DAO pool (as a percentage of the DAO pool). Watch the
 [Stake and Earn](/guides/dao-members/videos.md#stake-and-earn) video.
 
 1. Click the **_Stake_** button.
-   > The tokens available to stake depends on the number deposited and are
+   > The tokens available to stake depend on the number deposited and are
```
Comment (dcroote) [inline docs/guides/dao-members/staking.md:76]: I'd prefer this be 'The number of tokens available to stake depends'

**Review by dcroote — CHANGES_REQUESTED** [review]
One minor request, otherwise looks great, thanks

File: docs/guides/dao-members/staking.md (hunk 1 above)
Comment (omahs) [reply docs/guides/dao-members/staking.md]: Hey sure, just updated @dcroote

**Review by dcroote — APPROVED** [review]
Awesome, thank you so much

**Pull request merged**